modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -1,339 +1,425 @@
+* *Object store*
++
+Runtime Fabric does not support Object Store.
 
-Create an application properties file named *mule-app.properties* in the `src/main/app` folder of your project. When using the properties tab on CloudHub or Pivotal Cloud Foundry, these properties are overriden. See <<Managing Properties>> to see how these are loaded with values in each case.
+* *Scheduling*
++
+The scheduling feature is not available for Runtime Fabric deployments.
+To schedule tasks, use the Scheduler Endpoint element in your flows.
++
+See xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint].
 
-== Using Different Environments
 
-The Anypoint Platform enables you to handle separate xref:access-management::environments.adoc[Environments], such as production, QA, Dev, or any other custom one you may want to create. You can set some of these to be _sandbox_ environments, which allows you the freedom to test and experiment away from the public eye.
+* *Security updates*
++
+When a security update is available for the runtime version for an app, you can see and apply the update in Runtime Manager.
 
-Regardless of the deployment strategy used, deployments are always done onto a specific environment. Each manages a different set of deployments, and accessing each requires a different set of xref:access-management::managing-permissions.adoc[permissions], allowing you to divide roles clearly between the different teams of your organization.
+* *Troubleshooting*
++
+Runtime Fabric does not include the Insight troubleshooting tool. 
+Instead, use the command-line tools provided to obtain the health status of the cluster and components.
++
+See xref:runtime-fabric::troubleshoot-guide.adoc[Troubleshooting Guide for Runtime Fabric].
 
-Once an application has been tested in a sandbox environment and is ready for production, it can be directly promoted to a production environment, without needing to upload the application again. See  xref:deploying-to-your-own-servers.adoc#from-sandbox[deploying to your own servers] or xref:deploying-to-cloudhub.adoc#from-sandbox[deploying to cloudhub] to see how to do this.
 
-== Legacy Alternatives
+== Shared Resource Support for On-Premises App Deployments
 
-You can also deploy and manage your applications to Mule runtimes through other methods that exist from before there was an Anypoint Platform. These methods are still currently supported, but no new features are being added to them:
+When deploying on-premises, you can create `Domain` mule projects with no flows and a set of global configuration elements to share among other apps deployed to the same server.
```

Comment:
Should clarify this is for "hybrid" only. This does not apply for Runtime Fabric deployments.